### PR TITLE
Permissions checks implemented for get_item and delete_item

### PR DESCRIPTION
**Pull request by radugroza**

### CODE REVIEW

**Comment by marco-c** [comment]
Can you update the PR?

File: lib/class-wp-rest-plugins-controller.php [hunk 1]
```diff
@@ -41,11 +41,11 @@ public function register_routes() {
 	 * Check if a given request has access to read /plugins.
 	 *
 	 * @param  WP_REST_Request $request Full details about the request.
-	 * @return WP_Error|boolean
```
Comment (danielbachhuber) [inline lib/class-wp-rest-plugins-controller.php]: We should actually be returning `WP_Error` on failure as much as possible, as it gives us more descriptive errors:

image

**Comment by radugroza** [comment]
updated.

**Comment by marco-c** [comment]
The tests are failing.

**Comment by radugroza** [comment]
they pass now

**Pull request merged**